piksi_tools/console/settings.yaml

```diff
@@ -1539,14 +1539,26 @@
   name: cog_output_min_speed
   expert: true
   type: float
-  digits: '1'
   units: Meters per second
+  digits: '2'
```

Comment:
@john-michaelburke turns out the `digits` here is actually what controls the number of display digits in the settings table so I added it back in :shrug:

Reply:
That's what I was thinking.